plutus-core/test/Evaluation/Golden/polyErrorInst.plc.golden

```diff
@@ -1,2 +1,3 @@
 (Left An error has occurred:  User error:
-The provided Plutus code called 'error'.)
\ No newline at end of file
+The provided Plutus code called 'error'.
+Caused by: (error a_0))
```

Comment:
I admit the error is rather useless. Although if you have multiple calls to `error` in a program, having an uninstatiated type information is better than nothing.

@michaelpj @kwxm are we going to drop the typed CEK machine some day? I really dislike those uninstated types (and updating two machines all the time). In my view the typed CEK machine is completely redundant and can be dropped right away.

Reply:
I think we could probably drop the typed CK machine. We have a *lot* of machines at the moment, especially if we're duplicating them for TPLC and UPLC.